core/src/main/proto/spine/core/event.proto

```diff
@@ -99,6 +99,13 @@ message EventContext {
         RejectionContext rejection_context = 7 [(required) = true];
     }
 
+    // An ID of the root command, which lead to this event.
+    //
+    // If the `Event` is a reaction to another `Event`, then this attribute holds the identifier 
+    // of the very first command in this chain.
+    //
+    google.protobuf.Any root_command_id = 9 [(required) = true];
```

Comment:
`CommandId` is the built-in type. It's not a user-defined type, like, for example, an aggregate ID.

Why do we need `Any` here?